tests/Unit/Integrations/Route.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace DDTrace\Tests\Unit\Integrations;
+
+#[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD)]
+class Route
```

Comment:
This file was taken from Symfony 6.2. I haven't found another approach rather than copy to have it on our unit tests

Reply:
Sorted